api/core/v1alpha1/device_types.go

```diff
@@ -122,6 +163,44 @@ type DeviceStatus struct {
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
 }
 
+type ProvisioningInfo struct {
+	StartTime  metav1.Time `json:"startTime"`
+	EndTime    metav1.Time `json:"endTime,omitempty"`
+	RebootTime metav1.Time `json:"reboot,omitempty"`
```

Comment:
```suggestion
	EndTime    metav1.Time `json:"endTime,omitzero"`
	RebootTime metav1.Time `json:"reboot,omitzero"`
```
Should use `omitzero` as `metav1.Time` is a struct.